pages/curriculum.tsx

```diff
@@ -40,8 +53,16 @@ const Curriculum: React.FC = () => {
     return (
       <Layout>
         <div className="row mt-4">
-          <div className="col-9">{sortedLessons}</div>
-          <div className="col-3"></div>
+          <div className="col-8">{sortedLessons}</div>
+          <div className="col-4">
+            <ProgressCard progressCount={0}></ProgressCard>
```

Comment:
If you don't have children it's a common practice to write it with one self closing tag `<ProgressCard progressCount={0} />`

Reply:
Fixed 👍